### PR TITLE
Feat/trends page

### PR DESCRIPTION
**Pull request by VladislavPokhyl**
https://trello.com/c/O8YwAY75/48-trands-page
https://trello.com/c/3PYST5eZ/50-move-three-dots-modal-to-separate-component

### CODE REVIEW

**Comment by Mykyta-Pysarev** [comment]
Please, add screenshot of the layout.

**Comment by ArtemLigerko** [comment]
Cant`t find this branch in remote branchs. Is this from backend rep?

**Comment by VladislavPokhyl** [comment]
image

File: src/components/TrendItem.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          {trend.categoryName} · {trend.categoryValue}
+        </p>
+        <p className="pb-1 fw-bold">{trend.contentRefName}</p>
+        <span className="pb-1 text-secondary">Твітів: </span>
+        <span className="pb-1 text-secondary">{trend.tweetsCount}</span>
+      </Container>
+      <ContextMenu
+        contextItems={[
+          {
+            itemId: trend.id,
```
Comment (alexandrtovmach) [inline src/components/TrendItem.tsx]:
```suggestion
            itemId: `context_menu_trend_${trend.id}__interested`,
```

**Review by alexandrtovmach — APPROVED** [review]

**Review by alexandrtovmach — APPROVED** [review]

**Pull request merged**